src/Microsoft.AspNet.Razor.Runtime/IBufferedStringCollection.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Razor.Runtime
+{
+    /// <summary>
+    /// Contract for a buffered string collection.
+    /// </summary>
+    public interface IBufferedStringCollection
```

Comment:
Will this and its corresponding implementation be used for anything non-TagHelper related? The implementation of this guy for the most part just calls through to the `BufferEntryCollection`.